shopinvader_wishlist/services/wishlist.py

```diff
@@ -494,12 +489,15 @@ def _replace_items(self, record, params):
         return set_lines
 
     def _prepare_item(self, record, params):
-        return {
+        vals = {
             "product_set_id": record.id,
             "product_id": params["product_id"],
             "quantity": params.get("quantity") or 1,
-            "sequence": params.get("sequence") or 0,
         }
+        if "sequence" in params:
+            # Set sequence only if explicitly given
+            vals["sequence"] = params.get("sequence")
```

Comment:
From my understanding, the validator will set 0 as a sequence by default, so sequence should always be in the params, therefore you should never enter in that block.
Am I missing something?

Reply:
this is why in the same change I've removed that default :wink: